solution/Makefile

```diff
@@ -6,14 +6,44 @@ PWD := $(shell pwd)
 BUILD_ROOT ?= $(PWD)/_build
 ISO_ROOT ?= $(BUILD_ROOT)/root
 
+# Metadata
+PRODUCT_NAME := Zenko
+PRODUCT_LOWERNAME := zenko
+BUILD_TIMESTAMP := $(shell date -u +"%Y-%m-%dT%H:%M:%SZ")
+BUILD_HOST := $(shell hostname)
+
+ifneq '$(RELEASE)' '1'
+DEVELOPMENT_RELEASE = 1
```

Comment:
Maybe you don't need the variable `DEVELOPMENT_RELEASE` and could use the value of `RELEASE` directly.

Reply:
I copied this from the metalk8s example